zephyr/rz/rz_cfg/fsp_cfg/bsp/rzg3s/bsp_mcu_family_cfg.h

```diff
@@ -0,0 +1,59 @@
+/*
+* Copyright (c) 2020 - 2024 Renesas Electronics Corporation and/or its affiliates
+*
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#ifndef BSP_MCU_FAMILY_CFG_H_
+#define BSP_MCU_FAMILY_CFG_H_
+#define BSP_MCU_GROUP_RZG3S (1)
+#define BSP_CURRENT_CORE    (RZG3S_CORE_CM33)
+#define BSP_CORTEX_VECTOR_TABLE_ENTRIES    (16U)
+#define BSP_VECTOR_TABLE_MAX_ENTRIES       (496U)
+#include "bsp_mcu_device_pn_cfg.h"
+#include "bsp_mcu_device_cfg.h"
+#include "bsp_multi_core.h"
+#include "bsp_override.h"
+#include "bsp_mcu_info.h"
+#include "bsp_clock_cfg.h"
+
+#if defined(_RZG3S_TZ_SECURE)
+            #define BSP_TZ_SECURE_BUILD           (1)
+            #define BSP_TZ_NONSECURE_BUILD        (0)
+            #elif defined(_RZG3S_TZ_NONSECURE)
+            #define BSP_TZ_SECURE_BUILD           (0)
+            #define BSP_TZ_NONSECURE_BUILD        (1)
+            #else
+#define BSP_TZ_SECURE_BUILD           (0)
+#define BSP_TZ_NONSECURE_BUILD        (0)
+#endif
```

Comment:
@binhnguyen2434 : Please recheck coding style.

Reply:
@thaoluonguw ,  I have formatted the code.